tests/integration/test_export_inference.py

```diff
@@ -0,0 +1,253 @@
+        "--deterministic",
+        "True",
+        *DATASET[task]["overrides"],
+    ]
+
+    with patch("sys.argv", command_cfg):
+        main()
+
+    ckpt_files = list((tmp_path_train / "outputs" / "checkpoints").glob(pattern="epoch_*.ckpt"))
+    assert len(ckpt_files) > 0
+
+    # 2) otx test
+    tmp_path_test = tmp_path / f"otx_test_{model_name}"
+    command_cfg = [
+        "otx",
+        "test",
+        "--config",
+        recipe,
+        "--data_root",
+        DATASET[task]["data_root"],
+        "--engine.work_dir",
+        str(tmp_path_test / "outputs"),
+        "--engine.device",
+        fxt_accelerator,
+        *DATASET[task]["overrides"],
+        "--checkpoint",
+        str(ckpt_files[-1]),
+    ]
+
+    with patch("sys.argv", command_cfg):
+        main()
+
+    # 3) otx export
+    format_to_ext = {"OPENVINO": "xml"}  # [TODO](@Vlad): extend to "ONNX": "onnx"
+
+    tmp_path_test = tmp_path / f"otx_test_{model_name}"
+    for fmt in format_to_ext:
+        command_cfg = [
+            "otx",
+            "export",
+            "--config",
+            recipe,
+            "--data_root",
+            DATASET[task]["data_root"],
+            "--engine.work_dir",
+            str(tmp_path_test / "outputs"),
+            *DATASET[task]["overrides"],
+            "--checkpoint",
+            str(ckpt_files[-1]),
+            "--export_config.export_format",
+            f"{fmt}",
+        ]
+
+        with patch("sys.argv", command_cfg):
+            main()
+
+        assert (tmp_path_test / "outputs").exists()
+        assert (tmp_path_test / "outputs" / f"exported_model.{format_to_ext[fmt]}").exists()
+
+    # 4) infer of the exported models
+    task = recipe.split("/")[-2]
+    tmp_path_test = tmp_path / f"otx_test_{model_name}"
+    if "_cls" in recipe:
+        export_test_recipe = f"src/otx/recipe/classification/{task}/openvino_model.yaml"
+    else:
+        export_test_recipe = f"src/otx/recipe/{task}/openvino_model.yaml"
+    exported_model_path = str(tmp_path_test / "outputs" / "exported_model.xml")
+
+    command_cfg = [
+        "otx",
+        "test",
+        "--config",
+        export_test_recipe,
+        "--data_root",
+        DATASET[task]["data_root"],
+        "--engine.work_dir",
+        str(tmp_path_test / "outputs"),
+        "--engine.device",
+        "cpu",
+        *DATASET[task]["overrides"],
+        "--model.model_name",
+        exported_model_path,
+    ]
+
+    with patch("sys.argv", command_cfg):
+        main()
+
+    assert (tmp_path_test / "outputs").exists()
+
+    out, _ = capfd.readouterr()
+    assert TASK_NAME_TO_MAIN_METRIC_NAME[task] in out
+    torch_acc, ov_acc = tuple(re.findall(rf"{TASK_NAME_TO_MAIN_METRIC_NAME[task]}\s*│\s*(\d+[.]\d+)", out))
+    torch_acc, ov_acc = float(torch_acc), float(ov_acc)
+
+    msg = f"Recipe: {recipe}, (torch_accuracy, ov_accuracy): {torch_acc} , {ov_acc}"
+    log.info(msg)
+
+    _check_relative_metric_diff(torch_acc, ov_acc, 0.1)
```

Comment:
If torch_acc is almost 0 (including 0), and ov_acc is not 0, it mostly fails. How about adding criteria to deal whit this case? For example, if torch_acc - ov_acc < 0.01, then consider it ok.

Reply:
In general, the fact that `torch_acc = 0` is not ok, we should not reflect that workaround in criteria, which is fine for normal cases. I observe an improvement of training accuracy after deterministic is enabled, could you please check once again. If that wouldn't work, we can apply simplified criterion to faulty models, or increase the number of training epochs for them.

Reply:
I agree with you but I have a concern that it goes beyond integration test if it requires proper training. How about adding a test to compare scores in e2e and regression test when we start to add that not now?

Reply:
If we don't compare scores here at all, export and inference most likely fell apart after several PRs. Status of regression tests is almost always unknown (from the previous experience), also, they are designed to be run weekly, while we need a way to perform sanity check in ad-hoc manner.  
If train + inference fails to pass a sanity check, that looks like a problem. As I said, we can turn of this tests for the models that show 0 training accuracy.

Each case when OTX 2.0 fails to pass training sanity check should be addressed @sungmanc @vinnamkim @harimkang please share also your opinions.

Reply:
I agree with your opinion. Maybe we can find a way to check it's sanity within reasonable time. Let's look in more detail later.